### ISSUE TITLE
cfn-lint could not recognise DisableApiStop property in AWS::EC2::LaunchTemplate

### ISSUE THREAD

**Issue by LockonS** [aws-cloudformation/cfn-lint#2938]
### CloudFormation Lint Version

0.83.1

### What operating system are you using?

MacOS, Ubuntu

### Describe the bug

Reference: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-launchtemplate-launchtemplatedata.html#cfn-ec2-launchtemplate-launchtemplatedata-disableapistop

`DisableApiStop` was a feature flag released around May 24, 2022, which add the stop protection to EC2 instance, and currently couldn't be recognised by cfn-lint. 

The error reported now is shown below:

```
E3002 Invalid Property Resources/EC2Instance/Properties/DisableApiStop
```

### Expected behavior

cfn-lint should report no error with this property.

### Reproduction template

Part of the example configuration, for example the `DisableApiTermination` is well recognised but `DisableApiStop` is not recognised.

```yaml
AWSTemplateFormatVersion: "2010-09-09"

Resources:
  EC2Instance:
    Type: 'AWS::EC2::Instance'
    Properties:
      InstanceType: !Ref InstanceType
      DisableApiTermination: !Ref TerminationProtection
      DisableApiStop: !Ref StopProtection
```

**Comment by kddejong** [comment]
You're doc link points to `AWS::EC2::LaunchTemplate` and your example is for `AWS::EC2::Instance`. Looking at the [docs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ec2-instance.html) I don't see `DisableApiStop` for this resource type. 

In the CloudFormation [specs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-resource-specification.html) this parameter is available for `AWS::EC2::LaunchTemplate`
